src/app/job/job.component.html

```diff
@@ -53,7 +53,7 @@ <h5>{{job?.name}}</h5>
               <dt>Log output:</dt>
             </dl>
             
-            <div class="log-wrap" [ngClass]="{ 'logexpand' : logexpand }">
+            <div class="log-wrap">
```

Comment:
Do we want to have the `Expand logs` button at some point?
For now I'm ok with them already expanded.

Reply:
Logexpand is depreciated, with the change from showing many logs on one page, to each log being in it's own view. Now each log is expanded by default. Removing this old code